Kim Edits (#50)
mulesoft/docs-partner-manager: modules/ROOT/pages/endpoints.adoc

> There are three types of endpoints within AnyPoint Partner Manager

Reviewer: The information in this topic is presented in a really confusing way, because it is artificially separated into a definition list, a table, and a section comprising only cross-refs to further information...and each of these three separate parts overlap and repeat each other.

The title is Endpoint **Types** and **Protocols**. 

However, the majority of the narrative focuses on endpoint **types** (receive, target, acknowledgment...although the table later shows endpoint types as being send and receive...).

Then there is a table of **Names (Protocol or Type)**; **Protocols** (ASF2, SFTP, and HTTPS; and **Types** (send, receive).

Finally, a section of xrefs to sections about **protocol** settings ASF2, SFTP, and HTTPS.

Do you see what I am getting at about overlap and confusion?

Perhaps this topic should be revised to have three subtopics, one for each of Receive, Target, and Acknowledgement, and then those subtopics can include the appropriate protocols and actions (send, receive)?

Reviewer: The remainder is copyedited only.

Author: This file had partials, but it turns out the PM would like them moved into files. This should help minimize the confusion.

> ** Receive endpoints - On the Host side, for an inbound transmission you need endpoints your organization uses to receive transmissions from your partners.

Reviewer: Reformat all such list entries so that the bullet item is one fragment or sentence max, with the rest below.
You can also use definition list format: see the style reference.
```suggestion
** Receive endpoints
As host, your organization has external endpoints that it uses to receive transmissions from your partners.
```

Author: Rewritten with you edits in mind.

> These endpoints are resources your organization exposes to your trading partners.

Reviewer: Is this sentence necessary? Seems implied by the previous.

Author: Leaving it for now.

> ** Target endpoints - As the Host, your organization has internal endpoints where you route messages from the trading partner, after they are transformed within the message flow.

Reviewer: ```suggestion
** Target endpoints
As host, your organization has internal endpoints to which it routes messages from the trading partner, after those messages are transformed within the message flow.
```

Author: Fixed.

> ** Acknowledgement endpoints - Your partners provide you with endpoint configurations that can be used to send acknowledgement messages like 997s or 999s back to them.

Reviewer: ```suggestion
** Acknowledgement endpoints
Your partners provide you with their own external endpoint configurations, to which your organization can send acknowledgments, such as like 997 or 999 messages.
```

Author: Fixed.

> Acknowledgements are typically sent back to trading partners and correspond to one or more B2B messages you receive from them.

Reviewer: ```suggestion
Acknowledgments typically correspond to one or more B2B messages that you receive from them.
```

Author: Gonna leave as is for now.

> Refer to the following section for the type of endpoint you are configuring.

Reviewer: The following table defines endpoint name, protocol, and type settings.

Author: Leaving as is for now.

> 

Reviewer: Line 18
Omit subtitle

Reviewer: FORMAT:
Fix the tagging so that items like "Required: Yes" and "Default: None" appear on one line instead of two.

Author: Fixed.

> Default:: Auto-populates based on the following values:

Reviewer: ```suggestion
Default: Automatically populates with either Protocol or Type.
```

Author: Fixed.

> 

Reviewer: Suggest incorporating the following list into the previous sentence, as above.

Reviewer: Please implement these suggestions, should you agree with them, throughout the rest of the table. I will not repeat them.

Author: Fixed.

> * <<AS2 Receive Settings>>

Reviewer: Lines 75-83
These are more like items that should be in See Also links.
They could also be mentioned in a previous narrative about protocol types.

Author: These were partials that were not working right for me. As it turns out, we are converting them into files.
